src/DynamoPackages/Interfaces/IPackageLoader.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using Dynamo.PackageManager;
+
+namespace DynamoPackages.Interfaces
+{
+    /// <summary>
+    /// This interface is for PackageLoader. It can be expanded if needed,
+    /// currently only exposed inViewLoadedParams and referenced in WorkspaceDependencyViewExtension
```

Comment:
this seems like a strange comment to show to users.

Reply:
See my comments below